### PR TITLE
P4rtc support

### PR DESCRIPTION
**Pull request by badhrinathpa**
P4RT - a. introduce interface + refactor bess code to use interface b. add P4RT implementation of the interface

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -134,6 +134,10 @@ RUN CGO_ENABLED=0 go build -mod=vendor -o /bin/pfcpiface
 
 # Stage pfcpiface: runtime image of pfcpiface toward SMF/SPGW-C
 FROM alpine AS pfcpiface
+COPY conf /opt/bess/bessctl/conf
+COPY conf/p4info.bin /bin/
+COPY conf/p4info.txt /bin/
+COPY conf/bmv2.json /bin/
```
Comment (krsna1729) [inline Dockerfile:140]: group these as p4 directory in conf

**Comment by krsna1729** [comment]
Confirm that the refactor is based on latest code in master. There is too much bess specific code left in upf.go. Name the interface as fastpath or upf and put it in its own file.

**Comment by krsna1729** [comment]
The intent behind saying a. and b. was the commits to be split in such way to make it easier to review.

**Comment by badhrinathpa** [comment]
raised different pull request. Closing this

**Pull request closed without merging**